### ISSUE TITLE
Accuracy in the somethingv1 dataset

### ISSUE THREAD

**Issue by anti1990** [mit-han-lab/temporal-shift-module#51]
Can the test results on the somethingv1 dataset and the hyperparameter Settings achieve the accuracy of 47.3% in the paper? Num - segments = 8? Epoch25 isn't enough, is it? My 25 epochs are only 45.98 percent

Python main.py something RGB \

--arch resnet50 --num_segments 8 \

--gd 20 --lr 0.001 --lr_steps 10 20 --epochs 25 \

--batch-size 1-j 16 --dropout 0.5 --consensus_type=avg --eval-freq=1 \

- shift - shift_div = 8 -- shift_place = blockres \

-- tune_from = pretrained/TSM_kinetics_RGB_resnet50_shift8_blockres_avg_segment8_e50. PTH

**Comment by tonylins** [comment]
Hi, you should follow the hyper-parameter in the paper by using longer training epochs, a larger batch size, and a larger learning rate. Try something like:

python main.py something RGB \
     --arch resnet50 --num_segments 8 \
     --gd 20 --lr 0.025 --wd 1e-4 --lr_steps 20 40 --epochs 50 \
     --batch-size 64 -j 32 --dropout 0.5 --consensus_type=avg --eval-freq=1 \
     --shift --shift_div=8 --shift_place=blockres --npb

**Comment by anti1990** [comment]
Thank you&nbsp;&nbsp;for your reply.Thank you very much
  

 

 ------------------&nbsp;原始邮件&nbsp;------------------
  发件人:&nbsp;"Ji Lin"<notifications@github.com&gt;;
 发送时间:&nbsp;2020年1月3日(星期五) 中午11:54
 收件人:&nbsp;"mit-han-lab/temporal-shift-module"<temporal-shift-module@noreply.github.com&gt;;
 抄送:&nbsp;"秋月"<876746189@qq.com&gt;;"Author"<author@noreply.github.com&gt;;
 主题:&nbsp;Re: [mit-han-lab/temporal-shift-module] Accuracy in the somethingv1 dataset (#51)

 

 
Hi, you should follow the hyper-parameter in the paper by using longer training epochs, a larger batch size, and a larger learning rate. Try something like:
 
python main.py something RGB 
--arch resnet50 --num_segments 8 
--gd 20 --lr 0.025 --wd 1e-4 --lr_steps 20 40 --epochs 50 
--batch-size 64 -j 32 --dropout 0.5 --consensus_type=avg --eval-freq=1 
--shift --shift_div=8 --shift_place=blockres --npb

**Comment by Andy1621** [comment]
> Can the test results on the somethingv1 dataset and the hyperparameter Settings achieve the accuracy of 47.3% in the paper? Num - segments = 8? Epoch25 isn't enough, is it? My 25 epochs are only 45.98 percent
> 
> Python main.py something RGB \
> 
> --arch resnet50 --num_segments 8 \
> 
> --gd 20 --lr 0.001 --lr_steps 10 20 --epochs 25 \
> 
> --batch-size 1-j 16 --dropout 0.5 --consensus_type=avg --eval-freq=1 \
> 
> * shift - shift_div = 8 -- shift_place = blockres \
> 
> -- tune_from = pretrained/TSM_kinetics_RGB_resnet50_shift8_blockres_avg_segment8_e50. PTH

Hello, recently I have also tried to train TSM in sthv1, but I found that the pretrained model in README.md gets lower performance while testing. Could you please share me your pretrained model? Hopefully to your reply!

**Comment by YongkunZhan** [comment]
> Hi, you should follow the hyper-parameter in the paper by using longer training epochs, a larger batch size, and a larger learning rate. Try something like:
> 
> python main.py something RGB
> --arch resnet50 --num_segments 8
> --gd 20 --lr 0.025 --wd 1e-4 --lr_steps 20 40 --epochs 50
> --batch-size 64 -j 32 --dropout 0.5 --consensus_type=avg --eval-freq=1
> --shift --shift_div=8 --shift_place=blockres --npb

hello!I have doubts about the learning rate, which does not seem to be explicitly mentioned in the paper.
```
The training parameters for the
Kinetics dataset are: 100 training epochs, initial learning
rate 0.01 (decays by 0.1 at epoch 40&80), weight decay
1e-4, batch size 64, and dropout 0.5. For other datasets, we
scale the training epochs by half.
```

I want to know for sthv1/v2 the lr is 0.025 or 0.01?
